src/components/Card.js

```diff
@@ -22,15 +33,33 @@ export default class Card {
       ".card__trash-button"
     );
     this._cardTrashButton.addEventListener("click", () => {
-      this._handleTrashButton();
+      this._handleTrashClick();
     });
   }
 
+  setIsLiked(isLikedResp) {
```

Comment:
Needs correction: The card should not know anything about the api. Please rename the argument to something like `isLiked`

Reply:
corrected